### DISCUSSION TITLE
P110 does not have switch mode

### DISCUSSION THREAD

**Discussion by bedddo1929** [petretiandrea/home-assistant-tapo-p100#270] [Q&A] [answered]
Hi

I'm new to HA and this integration.  I've been running this for a while with my Tapo P110 plugs. Through the integration in HA I can see the sensors, but the plug also has a switch that can be controlled with the Tapo App and manually with the button on the side.  I've tried adding the plug in the configuration.yaml with the switch option, but it still just shows as a sensor. Is this just not provided in this integration or have I done something wrong.  Thanks

**Comment by petretiandrea** [comment, marked as answer]
Please refresh your page, the entities are added asynchronously
